src/tools/rbd/action/MirrorImage.cc

```diff
@@ -292,15 +313,20 @@ int execute_status(const po::variables_map &vm,
     formatter->dump_string("global_id", status.info.global_id);
     formatter->dump_string("state", state);
     formatter->dump_string("description", status.description);
+    formatter->dump_string("daemon_service_id", service_id);
+    formatter->dump_string("daemon_service_description", service_description);
     formatter->dump_string("last_update", last_update);
     formatter->close_section(); // image
     formatter->flush(std::cout);
   } else {
     std::cout << image_name << ":\n"
 	      << "  global_id:   " << status.info.global_id << "\n"
 	      << "  state:       " << state << "\n"
-	      << "  description: " << status.description << "\n"
-	      << "  last_update: " << last_update << std::endl;
+              << "  description: " << status.description << "\n";
+    if (!service_description.empty()) {
+    std::cout << "  service:     " << service_description << "\n";
```

Comment:
Nit: is the indent correct here?

Reply:
It was intentionally incorrect but to make the "  service:     " filed exactly under "  description: ", so I can be sure it has correct number of spaces (and will do if we will change this in future). But if you don't like this I will change.